cmake/SignMacOSApp.cmake

```
@@ -0,0 +1,15 @@
+# Script for macOS code signing
+
+if(NOT DEFINED APP_DIR)
+  set(APP_DIR "${CPACK_TEMPORARY_INSTALL_DIRECTORY}/ALL_IN_ONE")
+endif()
+
+file(GLOB LIBSTOSIGN "${APP_DIR}/GrandOrgue.app/Contents/Frameworks/*.*")
+foreach(LIBTOSIGN ${LIBSTOSIGN})
+  execute_process(COMMAND codesign --force --sign - "${LIBTOSIGN}")
+endforeach()
+execute_process(COMMAND codesign --force --sign - "${APP_DIR}/GrandOrgue.app/Contents/MacOS/GrandOrguePerfTest")
+execute_process(COMMAND codesign --force --sign - "${APP_DIR}/GrandOrgue.app/Contents/MacOS/GrandOrgueTool")
+execute_process(COMMAND codesign --force --sign - "${APP_DIR}/GrandOrgue.app")
```

```suggestion
execute_process(
  COMMAND codesign --force --sign -
  "${APP_DIR}/GrandOrgue.app/Contents/Frameworks/*.*" 
  "${APP_DIR}/GrandOrgue.app/Contents/MacOS/*"
)
execute_process(COMMAND codesign --force --sign - "${APP_DIR}/GrandOrgue.app")
```

Maybe it also works?

---

I tried this, it doesn't work. See #1835. I got
```
CPack: Executing pre-build script: /Users/grandorgue/Documents/GitHub/GrandOrgueBuild/build/osx/sign_package.cmake
/Users/grandorgue/Documents/GitHub/GrandOrgueBuild/build/osx/_CPack_Packages/macOS/DragNDrop/grandorgue-3.14.0-0.local.macOS.arm64/ALL_IN_ONE/GrandOrgue.app/Contents/Frameworks/*.*: No such file or directory
```